src/main.c

```diff
@@ -145,24 +300,39 @@ static void import() {
 
 	fseek(f, 0x100, SEEK_SET);
 	fread(spc, 65536, 1, f);
```

Comment:
There's no error checking on any of the calls to `fread` in this function, so it's possible to end up in a situation where the SPC and DSP byte arrays being parsed are made completely of uninitialized memory, if the file opened is small enough. And it ought to be easy enough to prevent:

```suggestion
	if (fread(spc, 65536, 1, f) != 1) {
		MessageBox2("Couldn't read ARAM of SPC file", "Import", MB_ICONEXCLAMATION);
		fclose(f);
		return;
	}
```

...or something similar. Ditto for similar occurrences. (I'm not sure if we care about checking for file writing errors (in fwrite/fclose) or seeking errors (in fseek), but those exist too.)

Reply:
Read verification has been added to SPC importing